### PR TITLE
768 - Create and deploy new release deployment cluster

### PR DESCRIPTION
**Pull request by shaunharrisonFR**
Add V1 files to repo

Issue: https://github.com/SecureApiGateway/SecureApiGateway/issues/768

### CODE REVIEW

File: v1/secure-api-gateway/sslcert/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+| Value    | description                          | default           |
+|------------------------------|-----------------------------------------------------------------------|-------------------|
+| externalCert.secretName      | Name of the kubernetes secret to store the sslcert                   | `sslcert`         |
+| externalCert.projectId       | GCP project that stores the secrets                                   | `example-project` |
+| externalCert.certPrefix      | The prefix of the cert to pull from google secrets manager . Use 'dev' to use the "dev" cert |dev|
+
+## Example
+
+The following will create the `bohocode-cdk` namespace, and use the cluster/certificate chart to deploy a secret to sslcert in `bohocode-cdk` namespace. It will deploy the the secrets called `bohocode-crt` and `bohocode-key` from google secret manager to tls.crt and tls.key in the `sslcert` secret.
```
Comment (dbadham-fr) [inline v1/secure-api-gateway/sslcert/README.md:16]: remove

File: v1/secure-api-gateway/sslcert/README.md
Comment (dbadham-fr) [inline v1/secure-api-gateway/sslcert/README.md:1]: Is the info contained here relevant for the new env?

File: v1/kustomize/overlay/7.2.0/defaults/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  # RCS connection settings for the RS API
+  RS_API_URI: http://securebanking-openbanking-uk-rs:8080
+  # Connection settings for the Consent Repo (hosted by IG)
+  CONSENT_REPO_URI: http://ig:80
+  RCS_CONSENT_RESPONSE_JWT_SIGNINGKEYID: rcs-jwt-signer
+  RCS_CONSENT_RESPONSE_JWT_ISSUER: secure-open-banking-rcs
+  # Realm to be used in Cloud Instance
+  AM_REALM: alpha
+  CERT_ISSUER: null-issuer
+  OB_ASPSP_ORG_ID: 0015800001041REAAY
```
Comment (dbadham-fr) [inline v1/kustomize/overlay/7.2.0/defaults/configmap.yaml:42]: Move to the overlay.

This is public information but it is specific to an FR deployment so shouldn't be the default value.

File: v1/secure-api-gateway/sslcert/README.md
Comment (shaunharrisonFR) [reply v1/secure-api-gateway/sslcert/README.md:1]: Removed whole read me file as not relevant info, can add in a proper readme later if required

File: v1/third-party/external-secrets/secretstore.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: external-secrets.io/v1beta1
+kind: SecretStore
+metadata:
+  name: gcp-store
+spec:
+  provider:
+    gcpsm:
+      projectID: sbat-dev
```
[the remark this answers is not in the document]
Comment (shaunharrisonFR) [reply v1/third-party/external-secrets/secretstore.yaml]: Changes to use values stored in the spec repo, will test to see if works once we can deploy again

File: v1/kustomize/overlay/7.2.0/defaults/configmap.yaml (hunk 2 above)
Comment (shaunharrisonFR) [reply v1/kustomize/overlay/7.2.0/defaults/configmap.yaml]: Done

**Review by BohoCode — COMMENTED** [review]
I think it looks good too. 

If we're going to manage this by tag rather than directory I think we should rename or remove the top level v1 directory now??

**Review by BohoCode — APPROVED** [review]

**Pull request merged**